bin/utils/getconfig.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#  Based on Texas Instruments Incorporated - http://www.ti.com/
+
+
+import logging
+import numpy as np
+import os
+import sys
+import yaml
+
+
+def read_file(model_dir):
+    try:
+        with open(model_dir + "param.yaml", "r") as file:
```

Comment:
please ask for the full file name, instead of assuming a name. This only limits the user and does not provide any significant benefit. For example, a user may have several config files and they may want to try one by one.